cmd/podman/manifest/rm.go

```diff
@@ -6,12 +6,14 @@ import (
 
 	"github.com/containers/podman/v5/cmd/podman/common"
 	"github.com/containers/podman/v5/cmd/podman/registry"
+	"github.com/containers/podman/v5/pkg/domain/entities"
 	"github.com/containers/podman/v5/pkg/errorhandling"
 	"github.com/spf13/cobra"
 )
 
 var (
-	rmCmd = &cobra.Command{
+	rmOptions = entities.ImageRemoveOptions{}
+	rmCmd     = &cobra.Command{
 		Use:               "rm LIST [LIST...]",
```

Comment:
need to add `[options]` between rm and LIST

Reply:
Good catch.